src/index.tsx

```diff
@@ -1,14 +1,49 @@
-import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import ReactDOM from 'react-dom';
+import { createHashRouter, RouterProvider } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { NotFoundRedirect } from './pages/NotFoundRedirect';
+import { NotFound } from './pages/NotFound';
+import { Navigate } from './pages/Navigate';
+import { PeoplePage } from './pages/PeoplePage';
 
-createRoot(document.getElementById('root') as HTMLDivElement)
-  .render(
-    <Router>
-      <App />
-    </Router>,
-  );
+const router = createHashRouter([
+  {
+    path: '/home',
+    element: <Navigate />,
+  },
+  {
+    path: '/',
+    element: <App />,
+    errorElement: <NotFoundRedirect />,
+    children: [
+      {
+        path: 'some/not/existing/page',
+        element: <NotFound />,
+      },
```

Comment:
It's cool that you decided to do a redirect for invalid urls! But if I understand your code correctly, it's supposed to open NotFoundComponent, but currently it's not loading
![image](https://github.com/mate-academy/react_people-table-basics/assets/130561076/4845b022-961c-4a5a-ba42-f04a5912aee2)


I am not sure how this path works 
path: 'some/not/existing/page',